fatiando/gridder.py

```diff
@@ -394,3 +395,55 @@ def cut(x, y, scalars, area):
               if x[i] >= xmin and x[i] <= xmax
               and y[i] >= ymin and y[i] <= ymax]
     return [x[inside], y[inside], [s[inside] for s in scalars]]
+
+
+def cut_regular(x, y, scalars, shape, area):
+    """
+    Return a subsection of a regular grid.
+
+    The returned subsection is not a copy! In technical terms, returns a slice
+    of the numpy arrays. So changes made to the subsection reflect on the
+    original grid. Use numpy.copy to make copies of the subsections and avoid
+    this.
+
+    Parameters:
+
+    * x, y
+        Arrays with the x and y coordinates of the data points.
+    * scalars
+        List of arrays with the scalar values assigned to the grid points.
+    * shape: tuple(ny,nx)
+        Shape of the original data points.
+    * area
+        ``(x1, x2, y1, y2)``: Borders of the subsection
+
+    Returns:
+
+    * ``[subx, suby, subscalars,shape_window]``
+        Arrays with x and y coordinates and scalar values of the subsection.
+    shape_window: tuple(ny,nx)
+        Shape of the cutted grid.
+
+    """
+    xmin, xmax, ymin, ymax = area
+    if xmin > xmax:
+        xmin, xmax = xmax, xmin
+    if ymin > ymax:
+        ymin, ymax = ymax, ymin
```

Comment:
@santis19 the comment for these 4 lines is the same as for #188 (use `assert` to raise an error)